src/app/home/pages/StoragesPage/components/AddEditStorageModal/__tests__/AddEditStorageModal.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer from '../../../../../../../reducers';
+
+import AddEditStorageModal from '../index';
+
+const store = createStore(rootReducer, {});
+
+describe('<AddEditStorageModal />', () => {
+  it('adds a S3 type storage', () => {
```

Comment:
Same nitpick as in #949, let's make sure these test names are clearly referring to just the form validation.